board/stm32h7/lladc.h

```diff
@@ -0,0 +1,49 @@
+// 5VOUT_S = ADC12_INP5
+// VOLT_S = ADC1_INP2
+#define ADCCHAN_VOLTAGE 2
+
+void adc_init(void) {
+  ADC1->CR &= ~(ADC_CR_DEEPPWD); //Reset deep-power-down mode
+  ADC1->CR |= ADC_CR_ADVREGEN; // Enable ADC regulator
+  delay(10000); // Let regulator to start
```

Comment:
is there a better way to do this?

Reply:
Better way to not use delay? Will check if there is some "busy" flag.